cognite_toolkit/_cdf_tk/utils.py

```diff
@@ -748,37 +719,6 @@ def verify_extraction_pipeline(self, external_id: str, skip_validation: bool = F
             )
         return -1
 
-    def verify_spaces(self, space: str | list[str]) -> list[str]:
```

Comment:
Removed this one. The reason is that verify_data_set/secutiry_categories etc are for replacing an externalId with an internal ID. This is not necessary for spaces. Thus this check makes cdf-tk deploy too strict as it then makes, for example, containers requiring READ access scoped to all.

Instead, in the deploy method there is a check on capabilities that is correctly scoped, such that for deploying a container we only need read and write access scoped to a specific dataset.